Source/model/draw/DropShadow.swift

```diff
@@ -1,16 +1,22 @@
 import Foundation
 
+public enum EffectSource: String {
```

Comment:
I don't recommend to use `:String` for public API, because it's parsing details and should be in the corresponding place.

Reply:
what if I make it internal?
`internal enum EffectSource: String`

Reply:
You can't make `EffectSource` internal because it used by public `DropShadow` class.

Reply:
Ok, let's keep `open enum EffectSource: String`